### ISSUE TITLE
Ask for pin name 

### ISSUE THREAD

**Issue by Vodianoi** [Vodianoi/QuickPingMod#19]

**Comment by DaveKX** [comment]
Hello! I don't know if you already have this on your todo, but it would be fun to be able to set the names of the pings when it's a useful resource (not really a problem not having it tho, you cant put it to the bottom of the list if you plan on add this xD). I just had this idea because I like to mark the spots on the map with acronyms like T for tin, C for copper etc

**Comment by Vodianoi** [comment]
Hi, thanks for the suggestion! I've actually already thought about this, and there are two ways we could implement it:

1. We could ask you for a name whenever you use the ping (e.g. with the G key), similar to how you add a pin on the map. This might be a bit intrusive while you're playing, but it would allow you to name each ping individually.

2. Alternatively, we could allow you to change the names of predefined objects (like Copper, Tin, Berries, etc.) in the settings. This would be less intrusive, but it would only allow you to change the names of certain objects, not all pins.

Let me know which option you prefer, or if you have any other suggestions!

**Comment by DaveKX** [comment]
I'd prefer the 2nd one!
However, and I don't know if this is possible, but if there could be a way to implement a mix of these two would be the best option, for example, we ping a copper deposit and it asks you for a name, and as I'm pointing to a 'useful resource' that being the copper deposit, it then would set the 'copper deposit' pin name to the name I wrote, in a way that when I ping a copper deposit again, it doesn't ask me a name anymore, but pin it on the map with the name I set before. This way the player could eventually change all the pins names. Although there would be a way to ask for a name again if the player wanted to change the name they set before, but this one could be a 'enable' option, that while enabled will always ask for a name and write this name over the previously name. If this is not possible to implement then I think the 2nd option is better, because as you've said, the 1st would be a bit intrusive.
